configure

```diff
@@ -1210,13 +1179,17 @@ case "${ARCH}" in
             ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} x86.o"
             ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} x86.lo"
 
+            check_avx2_intrinsics
```

Comment:
Maybe these could have () at the end? Makes it clear it is a function being called.

Reply:
Fixed.